dotnet3.5/S3/CopyObject/CopyObject.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
+// snippet-start:[S3.dotnet35.CopyObject]
```

Comment:
Maybe add a comment toward the top of the code file indicating what the following code does? This can help folks who open this file quickly know whether it will meet their needs or not.

Reply:
Added a comment inside the Program class that describes what the example does and also says which version of the SDK and .NET are used.